packages/lesmis/src/components/Tabs.js

```diff
@@ -3,25 +3,46 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  *
  */
-import React from 'react';
+import React, { forwardRef } from 'react';
 import PropTypes from 'prop-types';
 import MuiContainer from '@material-ui/core/Container';
 import styled from 'styled-components';
 import MuiTabs from '@material-ui/core/Tabs';
 import Skeleton from '@material-ui/lab/Skeleton';
 import MuiTab from '@material-ui/core/Tab';
+import { NAVBAR_HEIGHT } from '../constants';
 import { FlexStart } from './Layout';
 
 const LoadingTab = () => <Skeleton width={100} height={20} style={{ marginLeft: 30 }} />;
 
-export const TabBar = styled(MuiContainer)`
+const TabBarOuterContainer = styled.div`
```

Comment:
As far as I can tell this component is not needed and the styles could just be combined with `StickyTabBarOuterContainer`

Reply:
Yep, I was just aiming to keep the styling separate in case tabs are used elsewhere and don't want to be sticky at the bottom of the nav bar

Reply:
Happy to change if you think it's a case of YAGNI

Reply:
But your other suggestion is a much better way to do it - pull the sticky stuff fully out